app/locales/ur/overlap.json

```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "اوورلیپ چیک کریں",
+  "overlap_para_1": "سبز حلقے آپ کے مقامات کا گچھا ہیں ، بڑا کرنے کے لئے یہاں دبائیں۔ لال نشان آپ کے اندراج شدہ مقامات ہیں۔ \ n \ n بخوبی جامنی رنگ کے حلقوں کے بطور دکھائے جانے والے تصدیق شدہ مقدمات کی عوامی معلومات ڈاؤن لوڈ کرنے کے لئے بٹن دبائیں۔",
+  "show_overlap": "پبلک ڈیٹا چیک کریں",
```

Comment:
for show overlap, is this the line translated: "show_overlap": "Click to view the public dataset",?
(This line had gotten updated recently)